sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/WrapperSdkExceptionManager.java

```diff
@@ -118,5 +118,14 @@ private static File getFile(@NonNull UUID errorId) {
         String filename = errorId.toString() + DATA_FILE_EXTENSION;
         return new File(errorStorageDirectory, filename);
     }
+
+    /**
+     * Send an handled exception (used by wrapper SDKs).
+     *
+     * @param modelException An handled exception already in JSON model form.
+     */
+    public static void trackException(com.microsoft.azure.mobile.crashes.ingestion.models.Exception modelException) {
```

Comment:
Different name than iOS. `trackException` vs `trackModelException`?

Reply:
See my comment on ios PR. Android is fine.